vendor/github.com/gogo/protobuf/.travis.yml

```diff
@@ -1,10 +1,7 @@
-apt:
-  update: true
-
 env:
   - PROTOBUF_VERSION=2.6.1
   - PROTOBUF_VERSION=3.0.2
-  - PROTOBUF_VERSION=3.7.1
+  - PROTOBUF_VERSION=3.2.0
```

Comment:
Note, this seems to be a big step backward, but we were not aligned with what is used in k8s / openshift